pkg/framework/webhooks.go

```diff
@@ -19,12 +20,18 @@ var DefaultValidatingWebhookConfiguration = mapiv1.NewValidatingWebhookConfigura
 // DefaultMutatingWebhookConfiguration is a default mutating webhook configuration resource provided by MAO
 var DefaultMutatingWebhookConfiguration = mapiv1.NewMutatingWebhookConfiguration()
 
+// Webhooks may take up to 10 minutes to reconcile, as they could be just changed, but before we know about that,
+// a full Deployment (5 minutes) rollout and DaemonSet (another 5 minutes) rollout should occure. Then the
```

Comment:
I think this will read better like this, wdyt?
```suggestion
// Webhooks may take up to 10 minutes to sync. When changed we first have to wait for a full Deployment
// rollout (5 minutes) and DaemonSet rollout (another 5 minutes) to occur before they are synced. Then the
```